otherlibs/stdune-unstable/user_error.mli

```diff
@@ -16,6 +16,9 @@ module Annot : sig
 
     val to_dyn : payload -> Dyn.t
   end) : S with type payload = M.payload
+
+  (** The message has a location embed in the text. *)
+  module Has_embed_location : S with type payload = unit
```

Comment:
While going through the code, I was always stumbling when reading this name. Should it actually be `Has_embedded_location`?

Reply:
Indeed, I updated the name